### PR TITLE
Olwa changes

### PR DESCRIPTION
**Pull request by oliver-walker-hms**

### CODE REVIEW

File: plugin.rb [hunk 1]
```diff
@@ -50,9 +50,22 @@ def review_post(post)
                 if contains_restricted?(attachment)
                     links.push(attachment)
                     node = post_html.create_element 'p'# create paragraph element
+
```
Comment (TomKimsey) [inline plugin.rb]:
```suggestion
```

File: plugin.rb [hunk 2]
```diff
@@ -51,15 +51,15 @@ def review_post(post)
...
+                    attachment.replace '<font color="red">' + node + '</font>'
                     post.cooked = post_html
                     post.save!
                     hasUpdated = true
                 end
             end
 
             if hasUpdated 
-                updated_raw = ActionView::Base.full_sanitizer.sanitize(post.cooked)
+                updated_raw = post.cooked
```
Comment (TomKimsey) [inline plugin.rb]: Why is this assignment being done, can you just use post.cooked directly?

**Review by TomKimsey — CHANGES_REQUESTED** [review]

File: plugin.rb [hunk 3]
```diff
@@ -87,4 +87,4 @@ def contains_restricted?(attachment)
         @restricted_file_types = SiteSetting.file_attachment_whispers_file_extensions.split('|')
         @restricted_file_types.any? { |extension| attachment['href'].include?(extension)}
     end
-end
\ No newline at end of file
+end
```
Comment (TomKimsey) [inline plugin.rb:101]: looks like your editor touched the whitespace at the end too. just needs to be a separate commit.

File: plugin.rb (hunk 2 above)
Comment (oliver-walker-hms) [reply plugin.rb]: Actually that's a good point

Comment (oliver-walker-hms) [reply plugin.rb]: I'm not sure about the entire hasUpdated block of logic existing. It should probably just be a continuation of the last if statement

**Review by alexjhawk — APPROVED** [review]

**Review by TomKimsey — APPROVED** [review]
